content/c-sharp/concepts/enums/enums.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: 'Enums'
+Description: 'An Enum (or "Enumeration type") is a set of named labels that each represent a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Enums'
+  - 'Data Types'
+  - 'Arrays'
+  - 'Vectors'
+  - 'Pointers'
+  - 'Memory'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+An **Enum** (or "Enumeration type") is a set of named labels that each represent a number.
+
+## Syntax
+
+By default, each entry in an enum is assigned a zero-indexed value.
+
+```cs
+enum NinjaTurtle {
+  Leonardo, // 0
+  Michelangelo, // 1
+  Donatello, // 2
+  Raphael // 3
+}
+
+Console.WriteLine((int)NinjaTurtle.Raphael == 3); // prints "True"
+```
+
+However, specific number values can also be assigned.
+
+```cs
+enum BreadCount {
+  Dozen = 12,
+  BakersDozen = 13,
+  Gross = 144,
+  GreatGross = 1728
+}
+```
+
+The following example uses an enum to identify whether a given place is a city, country, or planet.
```

Comment:
Add an `## Example` header before this line.

Reply:
ah thanks, done!